Sources/SourceKitD/SourceKitD.swift

```diff
@@ -96,6 +100,24 @@ extension SourceKitD {
     logResponse(sourcekitdResponse)
 
     guard let dict = sourcekitdResponse.value else {
+      if sourcekitdResponse.error == .connectionInterrupted {
+        let log = """
+          Request:
+          \(req.description)
+
+          File contents:
+          \(fileContents ?? "<nil>")
+          """
+        let chunks = splitLongMultilineMessage(message: log)
+        for (index, chunk) in chunks.enumerated() {
+          logger.fault(
+            """
+            sourcekitd crashed (\(index + 1)/\(chunks.count))
+            \(chunk)
+            """
+          )
+        }
```

Comment:
If OSLog treats this as private (which I believe is the default?), will we just log multiple `sourcekitd crashed` lines with redacted chunks?

Reply:
Yes, we will. It’s a big unfortunate but I don’t have any better idea.